src/components/MasterChart/MasterChart.js

```diff
@@ -1,9 +1,17 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import Paper from 'material-ui/Paper';
+import { Grid, Row, Col } from 'react-bootstrap';
+import Toggle from 'material-ui/Toggle';
 import Dygraph from 'dygraphs';
+import _ from 'lodash';
 import DataParser from '../../utils/DataParser';
+import GraphTutorialPopover from '../GraphTutorialPopover/GraphTutorialPopover';
 import nodeInfo from '../../utils/Data/nodeInfo';
 import nodeInfo205 from '../../utils/Data/nodeInfo205';
+import './MasterChart.css';
+
+const GRAPH_HEIGHT = 320;
```

Comment:
May need to look at this value, and see if we can set graph height dynamically to cater for all screen sizes/resolutions.